### PR TITLE
made data table work with trace service

### PR DESCRIPTION
**Pull request by klein2ms**

### CODE REVIEW

File: src/visualization/visualization.js [hunk 1]
```diff
@@ -30,15 +30,16 @@ export class Visualization {
   
   subscribe() {
     let ea = this.eventAggregator;
-    let renderVisualization = this.renderVisualization;
+    let visualization = this;
     
-    ea.subscribe('onTraceChanged', payload => {
-      this.trace = payload.data;
-      renderVisualization();
+    ea.subscribe('traceChanged', payload => {
```
Comment (jbhilton11) [inline src/visualization/visualization.js]: We could use the TraceModel to take advantage of the event name extraction.
ie. traceModel.traceEvents.changed.event

This way, if for some reason the Exec Trace team wants to change the name of the event, we're not stuck doing a code change.

File: src/visualization/visualization.js [hunk 2]
```diff
@@ -30,15 +30,16 @@ export class Visualization {
...
+      console.log(payload.data);
+      visualization.trace = payload.data.trace;
+      visualization.renderVisualization();
     });
     
     ea.subscribe('onTraceFailed', payload => {
```
Comment (jbhilton11) [inline src/visualization/visualization.js]: We could use the TraceModel to take advantage of the event name extraction.
ie. traceModel.traceEvents.failed.event

This way, if for some reason the Exec Trace team wants to change the name of the event, we're not stuck doing a code change.

**Comment by jbhilton11** [comment]
As a general note, this technically solves feature-62, as it is using the trace information from the TraceService. We'll probably want to create another ticket for what I'm doing, which is adding the ability to select the expressions you want to see in the datatable.  I think this might have been something we talked about at a class you weren't there for.

Once the changes are made, I think we can merge this one into the master.

**Comment by klein2ms** [comment]
i was looking for something like this.  couldn't find it, but definitely good catch.  will make changes and commit them

**Comment by klein2ms** [comment]
Created a new issue to capture the additional functionality

**Pull request merged**